src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapper.java

```diff
@@ -663,26 +594,15 @@ Optional<float[]> getFloatsFromContext(ParseContext context, int dimension) thro
 
     @Override
     public ParametrizedFieldMapper.Builder getMergeBuilder() {
```

Comment:
I don't find any usage of getMergeBuilder , where it is getting used?

Reply:
in opensearch its an override. handled with mapping updates and also serialization. Basically a mapper is either constructed via parsing or from merging